hipify_cli.py

```diff
@@ -9,57 +9,65 @@
 import os
 import sys
 import argparse
+import json
 from hipify import hipify_python
 
-parser = argparse.ArgumentParser(
-    description='Top-level script for HIPifying, filling in most common parameters')
-parser.add_argument(
-    '--project-directory',
-    type=str,
-    default=os.getcwd(),
-    help="The root of the project. (default: %(default)s)",
-    required=False)
+def main():
+    parser = argparse.ArgumentParser(
+        description='Top-level script for HIPifying, filling in most common parameters')
+    parser.add_argument(
+        '--project-directory',
+        type=str,
+        help="The root of the project. (default: %(default)s)",
+        required=True)
 
-parser.add_argument(
-    '--output-directory',
-    type=str,
-    default=os.getcwd(),
-    help="The Directory to Store the Hipified Project",
-    required=False)
+    parser.add_argument(
+        '--output-directory',
+        type=str,
+        default=os.getcwd(),
```

Comment:
@pruthvistony: @micmelesse pointed out that we should be setting the default here to None, so that the if condition in https://github.com/ROCmSoftwarePlatform/hipify-torch/pull/3/files#diff-0b7bddf11afd4d0bda06f4377234786c0d9354f2c038e8dbd828dbba1837bcdaR58 doesn't get erroneously executed.

Reply:
done

Reply:
@micmelesse Once this PR is merged, the output_directory issue should be resolved for you